common/src/View/GamesPreferencePane.cpp

```diff
@@ -68,12 +72,25 @@ namespace TrenchBroom {
             m_stackedWidget = new QStackedWidget();
             m_stackedWidget->addWidget(m_defaultPage);
 
+            auto* glbShowUserConfigDirButton = new QPushButton(tr("Show user configurations..."));
+            connect(glbShowUserConfigDirButton, &QPushButton::clicked, this, &GamesPreferencePane::showUserConfigDirClicked);
+
+            auto* glbLayout = new QVBoxLayout();
```

Comment:
This derails the UI completely. I find the placement of the button below the list questionable, and it also doesn't fit very well with the rest of TB's UI. I'd have to think about it a bit to have a suggestion.